tests/github-action-helper.sh

```diff
@@ -143,6 +208,11 @@ wait_for_crd_to_be_ready_custom() {
       sleep 2
     done
 EOF
+  # timeout command return exit status 124 if command times out
+  if [ $? -eq 124 ]; then
```

Comment:
if possible move this to helper function so that we can avoid code duplication

Reply:
created helper function